digits/dataset/tasks/create_db.py

```diff
@@ -191,19 +189,19 @@ def process_output(self, line):
             if not hasattr(self, 'distribution') or self.distribution is None:
                 self.distribution = {}
 
-            self.distribution[match.group(1)] = int(match.group(2))
-
-            data = self.distribution_data()
-            if data:
-                socketio.emit('task update',
-                              {
-                                  'task': self.html_id(),
-                                  'update': 'distribution',
-                                  'data': data,
-                              },
-                              namespace='/jobs',
-                              room=self.job_id,
-                              )
+            self.distribution[match.group(1)] = [int(match.group(2)), 0]
```

Comment:
I find this a little bit difficult to read. Could you use a dict with two keys for example `{'count': int(match.group(2)), 'error_count':0}`?